tests/phpunit/tests/term/isObjectInTerm.php

```diff
@@ -167,13 +168,15 @@ public function test_should_not_be_fooled_by_a_stale_relationship_cache() {
 		$num_queries = $wpdb->num_queries;
 		$this->assertTrue( is_object_in_term( $o, 'wptests_tax', $terms[0] ) );
 		$num_queries++;
+		$num_queries++;
```

Comment:
```suggestion
		$num_queries += 2;
```

🔢 Borderline nit: it's cleaner code to do this on one line, but you know that ;)